migrations/20181011183056_add_recipe_table.js

```diff
@@ -0,0 +1,14 @@
+
+exports.up = function (knex, Promise) {
+  return knex.schema.createTable('recipe', function (tbl) {
+    // we must use the callback syntax for .createTable()
+    tbl.increments('recipe_id'); // pass the name if you wanted to be called anything other than id
+    tbl
+      .string('name', 255)
+      .notNullable()
+  });
+};
```

Comment:
There should be a column here that connects the recipe to the dish through a foreign key.